src/types/MetadataPropertyType.ts

```diff
@@ -0,0 +1,90 @@
+type MetadataProperty = {
+	key: string;
+	scope: string;
+	type: 'string' | 'date';
+	format?: string;
+	label: string;
+	'bmi-isNotEmpty'?: boolean;
+	'bmi-errorMessage'?: string;
+};
+
+export const MetadataProperties: MetadataProperty[] = [
```

Comment:
This is not a type and thus doesn't belong here.

Reply:
Moved to util